content/en/real_user_monitoring/browser/_index.md

```diff
@@ -1814,6 +1814,12 @@ A list of request origins ignored when computing the page activity. See [How pag
 **Type**: String<br/>
 URL pointing to the Datadog Browser SDK Worker JavaScript file. The URL can be relative or absolute, but is required to have the same origin as the web application. See [Content Security Policy guidelines][22] for more information.
 
+`compressIntakeRequests`
+: Optional<br/>
+**Type**: Boolean<br/>
+**Default**: `false`<br/>
+Compress requests sent to the Datadog intake to reduce bandwidth usage when sending large amount of data. The compression is done in a Worker thread, see [Content Security Policy guidelines][22] for more information.
```

Comment:
```suggestion
Compress requests are sent to Datadog for intake to reduce bandwidth usage when sending large amount of data. The compression is done in a Worker thread. [Content Security Policy guidelines][22] for more information.
```

Reply:
Thanks for your suggestion.

> Compress requests are sent to Datadog for intake to reduce bandwidth usage

This formulation seems a bit weird to me, I'm not sure to understand the phrase anymore. When this parameter is set to `true`, the Browser SDK will compress RUM data before sending it to Datadog. Was that unclear from my original suggestion?

Reply:
One suggestion -- add an "s" to "amount", so it would say "when sending large amount**s** of data."